DataProc/CPass0/checkResTree.C

```diff
@@ -9,10 +9,18 @@ void checkResTree(const char* fname = "ResidualTrees.root", const char* tname =
   TTree* tr = (TTree*)tf.Get(tname); // we want macro to crash if the tree is not there
   int nev = tr->GetEntries();
   int rdb = 0;
+  bool corrupted = kFALSE;
   for (int i=0;i<nev;i++) {
     rdb = tr->GetEntry(i);
     if (rdb<=0 || rdb>10000) {
       printf("CORRUPTED Residuals file, ev %d got %d bytes\n",i,rdb);
+      corrupted = kTRUE;
     } 
   }
+  if (corrupted) {
```

Comment:
If the tree is corrupted, it is not guaranteed to go through the all ```nev``` of the loop, if the messages of this ```if .. else ..``` in the log are to be used by the validation, it is better to add a ```break;``` after the ```corrupted = kTRUE;```

Reply:
Actually I added this only to have a "positive" message in the log if the file is not corrupted.

Reply:
I've checked with the script as you have it and the macro as
```
void checkResTree(const char* fname = "ResidualTrees.root", const char* tname = "delta")
{
  TFile tf(fname);
  TTree* tr = (TTree*)tf.Get(tname); // we want macro to crash if the tree is not there
  int nev = tr->GetEntries();
  int rdb = 0;
  bool corrupted = kFALSE;
  for (int i=0;i<nev;i++) {
    rdb = tr->GetEntry(i);
    if (rdb<=0 || rdb>10000) {
      printf("CORRUPTED Residuals file, ev %d got %d bytes, to be Killed\n",i,rdb);
      corrupted = kTRUE;
      break;
    } 
  }
  if (corrupted) {
    printf("The residual tree file is corrupted: job must be Killed\n");
  }
  else {
    printf("The residual tree file is OK\n");
  }
}
```
using existing ```/alice/validation/validation.sh```, it works both on the corrupted and on healthy ```ResidualTrees.root``` as expected.

Reply:
Ok, so only the break was added, right? I can add it sure.

Reply:
Also added "Killed" to messages signaling a corruption

Reply:
Ok, done. Can you check? If you think it is fine, I will merge the changes.
Thanks!